api/models/publication.js

```diff
@@ -31,7 +31,7 @@ const Publication = BaseModel.extend({
       id: attributes.id,
       url: attributes.url,
       title: attributes.title,
-      source: this.related('source').toJSON(),
+      source_id: this.attributes.source_id,
```

Comment:
This is a bit messy :/

`attributes` here is `this.toJSON()`. However, it'll only contain the model's visible attributes. `source_id` isn't one of them, so we need to get directly from `this.attributes`